src/openvic-simulation/politics/Government.hpp

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include "openvic-simulation/types/IdentifierRegistry.hpp"
+#include "openvic-simulation/politics/Ideology.hpp"
+
+namespace OpenVic {
+    struct GovernmentTypeManager;
+
+    struct GovernmentType : HasIdentifier {
+        friend struct GovernmentTypeManager;
+
+    private:
+        std::vector<std::string> ideologies;
+        const bool elections, appoint_ruling_party;
+        const uint16_t election_duration;
```

Comment:
```suggestion
        const Timespan election_duration;
```
`Timespan` represents a number of days - ask around and/or test the game to see if the defines value is in weeks or months and adjust accordingly.

Reply:
Election duration is determined by CAMPAIGN_DURATION in the defines, which is in months. It does not vary by government form.

Reply:
Ah ok, so it's not just that they all have `duration = 48`, even when that's changed for some they all have the same?

Reply:
duration is months between elections IIRC, and some mods do give it different values.

Reply:
@BrickPi `election_duration` should probably be renamed `term_duration`.